lib/features/cleanup/config.dart

```diff
@@ -27,6 +27,14 @@ library;
 import 'package:flutter/material.dart';
 
 import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:rattle/constants/spacing.dart';
+import 'package:rattle/providers/selected.dart';
+import 'package:rattle/providers/stdout.dart';
+import 'package:rattle/r/extract.dart';
+import 'package:rattle/r/source.dart';
+import 'package:rattle/utils/get_inputs.dart';
```

Comment:
This is misleading - are you using getInput or getIgnored - they should be in different files for self-explanation.